src/NotificationBundle/Tests/Twig/Components/NotificationIntegrationsTest.php

```
@@ -0,0 +1,488 @@
+        // Refresh component
+        $rendered2 = $component->refresh()->render()->toString();
+        $this->assertMatchesHtmlSnapshot($this->replaceChecksum($this->replaceUuid($rendered2)));
+
+        // Should still show the same content
+        self::assertStringContainsString('Refresh Test', $rendered2);
+    }
+
+    /**
+     * @dataProvider transportTypesProvider
+     */
+    public function testDifferentTransportTypes(string $transportName, string $transportType): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$transportType'. (undefined)

(UnusedFormalParameter)